e2e_tests/test_static_links.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from playwright.sync_api import expect
+
+
+@pytest.mark.parametrize(
+    "link_text, expected_url",
+    [
+        (
+            "The National Archives",
+            "https://www.nationalarchives.gov.uk/",
+        ),
+    ],
+)
+def test_header_link(page, link_text, expected_url):
+    """
+    Given an unauthenticated user
+    When they click a header link
+    Then the user is redirected to the associated page
+    """
+    page.goto("/")
+    page.click(f'text="{link_text}"')
+    expect(page).to_have_url(expected_url)
+
+
+SERVICE_NAME = "AYR – Access Your Records – GOV.UK"
```

Comment:
Does this need to be defined here?

Reply:
nope was used for some old assertions I removed. good catch